ui/src/SurveyCaseSearch.js

```diff
@@ -228,6 +228,17 @@ class SurveyCaseSearch extends Component {
             ":
           </Typography>
         )}
+        {this.state.caseSearchTerm &&
+          !this.state.isWaitingForResults &&
+          this.state.caseSearchResults.length === 100 && (
+            <Typography
+              variant="h6"
+              color="inherit"
+              style={{ marginTop: 10, marginBottom: 10 }}
+            >
+              Warning, there are too many search results to show
```

Comment:
Pedant here, but a slightly more accurate term is something like
"Search results are limited to 100, so there may be more matching cases"

if there was exactly 100 matches, then the 'too many thing is misleading.  Not a big issue